int16 lane_status
int16 lane_dist
int16 side_status
int16 side_dist
